components/filter/filter.js

```diff
@@ -118,7 +119,6 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 			.d2l-filter-dimension-set-value-text {
 				overflow: hidden;
 				text-overflow: ellipsis;
```

Comment:
If we're using the multi-line clamping instead of the single-line ellipsis, I _think_ we might be able to remove this `text-overflow: ellipsis`. I _think_ we still need the `overflow: hidden` however.

Reply:
If that's the case, I think we'd need to move `text-flow: ellipsis` into the `.d2l-filter-dimension-header-text` section so it's still set for the header.  It's still just a single-line ellipsis.

Reply:
Ok cool. If we still need it, no issue keeping it I think.